src/main/main.ts

```diff
@@ -149,6 +149,6 @@ export function main(argv_in: string[]) {
 }
 
 // only call main() if this is the main module
-if (typeof module !== 'undefined' && !module.parent) {
+if (typeof module !== 'undefined' && !module.children) {
```

Comment:
https://nodejs.org/api/deprecations.html#dep0144-moduleparent

this code change doesn't look quite right

Reply:
I read the nodejs module.children documentation and it returns the same object as module.parent. 
I will change this but how do i test the changes.

Reply:
I need some help

Reply:
#### I think this is the right way:
```typescript
 if (typeof module !== 'undefined' && require.main !== module) {
```

#### Local test way:

```shell
$ yarn run start --help
```
The correct output is:

<img width="957" alt="image" src="https://user-images.githubusercontent.com/8198408/161183104-a70297a7-a465-4999-b290-fdcd4bdf22d4.png">

As long as Fiddle opens properly after the change, then the change is correct.

Reply:
Thanks for help @BlackHole1

Reply:
Yes I can confirm The fiddle is running without any problems.